pkg/api/norman/customization/cluster/validator.go

```diff
@@ -84,71 +73,6 @@ func (v *Validator) Validator(request *types.APIContext, schema *types.Schema, d
 	return v.validateGKEConfig(request, data, &clusterSpec)
 }
 
-func (v *Validator) validateScheduledClusterScan(spec *mgmtclient.Cluster) error {
-	// If this cluster is created using a template, we dont have the version in the provided data, skip
-	if spec.ClusterTemplateRevisionID != "" {
-		return nil
-	}
-
-	// If CIS scan is not present/enabled, skip
-	if spec.ScheduledClusterScan == nil ||
```

Comment:
We need to remove references to CIS scan spec from the API. Check the cis scan related structs in types https://github.com/rancher/rancher/blob/release/v2.7/pkg/apis/management.cattle.io/v3/cluster_scan_types.go
Also fields in Cluster spec - https://github.com/rancher/rancher/blob/release/v2.7/pkg/apis/management.cattle.io/v3/cluster_types.go

Reply:
Removed the cis scan related references from the API.